nixos/modules/services/databases/influxdb.nix

```diff
@@ -68,7 +68,7 @@ let
 
     collectd = [{
       enabled = false;
-      typesdb = "${pkgs.collectd}/share/collectd/types.db";
+      typesdb = "${cfg.collectdPackage}/share/collectd/types.db";
```

Comment:
No strong opinion on it. Conceptually I think multiple-outputs is cleaner but not sure if there are any practical benefits in this case.

Is it possible to deploy `collectd` without the data? If so, that'd be a good case for moving the data to a separate output.  Another thing to consider is that without hardlinking optimization you'd waste quite a bit of space if both the data-only and the full collectd paths end up being referenced by a gc root.